EDSM_RSE_Database_Creator.py

```diff
@@ -148,17 +141,18 @@ def createDatabase(self):
     def isDatabasePresentAndValid(self):
         try:
             self.openDatabase()
+        except:
+            self.conn = None
+            self.c    = None
+            return False
+        try:
             # test if connection is really open
             self.c.execute("SELECT * from systems LIMIT 10")
             results = self.c.fetchall()
-            self.conn.close()
-            self.c = None
-            self.conn = None
             return len(results) > 0
         except:
-            self.c = None
-            self.conn = None
-        return False
+            self.conn.commit()
```

Comment:
Exception = no connection possible so this will fail and throw another exception